polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -705,6 +776,90 @@ $disabled-fade: 0.6;
   list-style: none;
 }
 
+.ListSecondary {
```

Comment:
This class name is only used when the se23 feature is enabled

Reply:
I usually still wrap the inside with se23 just in case but its fine! less clean up this way

Reply:
maybe nice to wrap it though so its obvious to the clean up team? we might try to clean up programatically